runner.go

```diff
@@ -212,6 +241,20 @@ func (r *Runner) GetStats() RunnerStats {
 	return *r.stats
 }
 
+// SetBackoffEnabled enables/disables runner back-off. This will increase time between runs (between the ceiling and floor)
+// based on the last run duration.
+func (r *Runner) SetBackoffEnabled(runIntervalFloor time.Duration, runIntervalCeiling time.Duration) {
+	r.runIntervalCeiling = runIntervalCeiling
+	r.runIntervalFloor = runIntervalFloor
+	r.intervalBackoffEnabled = true
+}
+
+// SetScheduledRebuildEnabled enables/disables a periodic complete view rebuild based on the given schedule instance.
+func (r *Runner) SetScheduledRebuildEnabled(rebuildSchedule *RebuildSchedule) {
```

Comment:
I would add a check so this can't be called twice. Like:

```go
if r.rebuildSchedule != nil {
    return
}
```

Also it can't be really disabled, can't it? It will keep running in the goroutine and never be garbage collected. At least until it blocks writing to the channel after the next iteration.